client/src/components/PaymentMethods/useStyle.ts

```diff
@@ -0,0 +1,35 @@
+import { Theme } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+
+export const useStyles = makeStyles((theme: Theme) => ({
+  card: {
+    display: 'flex',
```

Comment:
These styles can directly be added to the `Box` component as props

```
<Box display="flex alginItems="center"...> 
```

Reply:
Box only can take sx as props for CSS elements, so I think this way is easier for styling.